.dotnet/src/Custom/Chat/DeveloperChatMessage.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+
+namespace OpenAI.Chat;
+
+/// <summary>
+/// Represents a chat message of the <c>developer</c> role as supplied to a chat completion request. A developer message is
+/// generally supplied as the first message to a chat completion request and guides the model's behavior across future
+/// <c>assistant</c> role response messages. These messages may help control behavior, style, tone, and
+/// restrictions for a model-based assistant. Developer messages replace system messages for o1 models and newer.
+/// </summary>
```

Comment:
nit: Just like you added that sentence at the end about using developer messages with o1 models and newere, could you add the counterpart to the doc comment in `SystemChatMessage`?